src/OrbitQt/main.cpp

```diff
@@ -75,7 +76,8 @@ using Context = orbit_ssh::Context;
 Q_DECLARE_METATYPE(std::error_code);
 
 void RunUiInstance(const DeploymentConfiguration& deployment_configuration,
-                   const Context* ssh_context) {
+                   const Context* ssh_context, const QStringList& command_line_flags,
+                   const std::string& capture_file_path = "") {
```

Comment:
Since `orbit_qt::FileTarget(std::filesystem::path)` takes a path anyways:
```suggestion
                   const std::filesystem::path& capture_file_path = "") {
```

Reply:
Changed accordingly. Thanks!